src/modules/job-info/job_state.c

```diff
@@ -325,10 +327,13 @@ static void update_job_state_and_list (struct info_ctx *ctx,
     oldlist = get_list (jsctx, job->state);
     newlist = get_list (jsctx, newstate);
 
+    /* must call before job_change_list(), to ensure timestamps are
+     * set any timestamp sorting is done correctly
```

Comment:
Is there perhaps one word missing in this comment? Maybe `to ensure timestamps are set *before* timestamp sorting is done`-- (hm, still not sure that is right either)

Reply:
yes, a word is missing :-)